tools/HogEdit/HogFormat.h

```diff
@@ -39,39 +40,61 @@ FILE 1			[filelen(FILE 1)]
 FILE NFILES-1		[filelen(NFILES -1)]
 */
 
-inline std::array<char, 56> array_with(char val) {
+static inline std::array<char, 56> array_with(char val) {
   std::array<char, 56> arr;
   arr.fill(val);
   return arr;
 }
 
-struct HogHeader {
-  std::array<char, 4> tag = {'H', 'O', 'G', '2'}; // "HOG2"
-  uint32_t nfiles = 0;                            // number of files in header
-  uint32_t file_data_offset = 68;                 // offset in file to filedata.
-  std::array<char, 56> reserved = array_with(-1); // filled with 0xff
+struct /* [[gnu::packed]] */ HogHeader {
```

Comment:
What is purpose of commented [[gnu::packed]]?

Reply:
I wanted to pack the struct but due to how it's used, the compiled can't deal with it... at least at present. In the future it may be functional.